### PR TITLE
FEAT_iban: populate field from properties

### PR DESCRIPTION
**Pull request by emidev98**
## Summary
Populate the IBAN based on the **paymentMethodsConfiguration** **ownerName** and **ibanNumber** from **sepadirectdebit** property.

## Tested scenarios

[✔️] Fill the ownerName and ibanNumber
[✔️] Fill the ownerName 
[✔️] Fill the ibanNumber
[✔️] Leave it blank.

### CODE REVIEW

File: src/components/Sepa/components/IbanInput/IbanInput.js [hunk 1]
```diff
@@ -13,15 +13,30 @@ class IbanInput extends Component {
         this.state = {
             status: 'ready',
             data: {
-                'sepa.ownerName': '',
-                'sepa.ibanNumber': ''
+                'sepa.ownerName': props.ownerName ? props.ownerName : '',
+                'sepa.ibanNumber': props.ibanNumber ? props.ibanNumber : ''
```
Comment (pabloai) [inline src/components/Sepa/components/IbanInput/IbanInput.js:17]: Hey @emidev98, glad to see you contributing again!

A couple of comments here:
This `props.ibanNumber ? props.ibanNumber : ''` can be simplified like this `props.ibanNumber || ''`. We could as well use optional chaining like this `'sepa.ownerName': props?.ownerName`.

But also, to align with other components, I'd wrap both `ownerName` and `ibanNumber` in a `data` object.
```js
data: {
    'sepa.ownerName': props?.data?.ownerName,
    'sepa.ibanNumber': props?.data?.ibanNumber
}
```
We could as well include the properties only when necessary:
```js
data: {
    ...(props?.data?.ownerName && { 'sepa.ownerName': props.data.ownerName }),
    ...(props?.data?.ibanNumber && { 'sepa.ibanNumber': props.data.ownerName })
}
```
Keep in mind that tests will need to be updated as well.

**Review by pabloai — CHANGES_REQUESTED** [review]
Adding some comments and suggestions.

File: src/components/Sepa/components/IbanInput/IbanInput.js (hunk 1 above)
Comment (emidev98) [reply src/components/Sepa/components/IbanInput/IbanInput.js]: Make sense fro code standards inside the library.
Already did the change and wrote the tests

**Review by pabloai — APPROVED** [review]

**Comment by emidev98** [comment]
Dear Adyen development team, when I will have this feature available in npm repository? Because it's a very important business requirement for our product to improve the UX

**Comment by pabloai** [comment]
I'm merging this PR. This will be included in our next release which is planned for next Monday (Aug 31).
Thanks again for your contribution!

**Pull request merged**